code-challenges.test.js

```diff
@@ -35,10 +35,36 @@ const num2 = 0
 const num3 = -7
 // Expected output: "-7 is not divisible by three"
 
+describe("byThree", () => {
+    it("takes in a number as an argument and returns if the number is divisible by three or not", () => {
+        expect(byThree(num1)).toEqual("15 is divisible by three")
+        expect(byThree(num2)).toEqual("0 is divisible by three")
+        expect(byThree(num3)).toEqual("-7 is not divisible by three")
+    })
+})
```

Comment:
I often recommend placing the test variables being used inside the it block just before the expect statement so they are confined to the scope they are used in.